docs/articles/blog/2019-2-7-testcafe-v1-0-0-released.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: post
+title: TestCafe v1.0.0 Released
+permalink: /blog/:title.html
+---
+# TestCafe v1.0.0 Released
+
+Meet TestCafe v1.0.0, our first major update that includes some big features: video recording, configuration file, "live mode" for rapid test development and much more.
```

Comment:
Meet?

Reply:
big features?

Reply:
Well, this is a blog post, not a doc, so I thought that the opening phrase could be something like that... However, if we choose to go with something more neutral, we can write:

*TestCafe v1.0.0 is our first major update that includes features like video recording, configuration file, "live mode" for rapid test development and much more.*